reactor-core/src/test/java/reactor/core/publisher/FluxOnBackpressureBufferStrategyTest.java

```diff
@@ -464,30 +466,26 @@ public void fluxOnBackpressureBufferStrategyNoCallback() {
 		assertThat(hookCapturedError).as("unexpected hookCapturedError").isNull();
 	}
 
+	@ParameterizedTest
+	@ValueSource(ints = {-1, 0})
+	public void fluxOnBackpressureBufferStrategyRequiresPositiveMaxSize(int maxSize) {
```

Comment:
while you're improving the tests (nice 👍), you can remove the `public` modifier from the amended tests